vpr/src/route/router_lookahead_map.h

```diff
@@ -11,46 +11,31 @@ class MapLookahead : public RouterLookahead {
     explicit MapLookahead(const t_det_routing_arch& det_routing_arch, bool is_flat);
 
   private:
```

Comment:
Take a crack at Doxygen comments for all functions etc. here if you can .... use in the new VPR internals for router lookahead tab.
Also make sure that there is a high level description of the router lookahead (and the various versions) in that new Doxygen documentation.